### PR TITLE
Show token values in docs

### PR DESCRIPTION
**Pull request by anniehu4**
### Summary:
- Makes sure token values are showing in Doc pages:
Screen Shot 2022-05-02 at 4 48 35 PM

- Updates typography docs to show font size & line height for easier reference:
Screen Shot 2022-05-02 at 4 48 44 PM

### Test Plan:
Double check changes in Chromatic

### CODE REVIEW

File: .storybook/components/DesignTokens/Tier1/Colors.js [hunk 1]
```diff
@@ -27,13 +27,3 @@ export class Tier1Colors extends Component {
     );
   }
 }
-
-function filterTokens(prefix) {
-  return Object.entries(tokens)
-    .filter(([name]) => name.startsWith(prefix))
-    .map(([name, { value, comment }]) => ({
```
Comment (anniehu4) [inline .storybook/components/DesignTokens/Tier1/Colors.js:34]: this was expecting the wrong structure -- it should have been parsing for `[name, value]`. moved this into its own `util` function so I wouldn't have to fix it everywhere

File: .storybook/components/DesignTokens/Tier1/TypographyPresets.js [hunk 2]
```diff
@@ -1,161 +1,60 @@
 import React, { Component } from 'react';
 import { Grid } from '../../../../src';
-import styles from '../../../../src/components/Utilities/TypographyPresets.module.css';
```
Comment (anniehu4) [inline .storybook/components/DesignTokens/Tier1/TypographyPresets.js:3]: not sure what happened here but utility classes shouldn't be in module files

File: .storybook/util/filterTokens.ts [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import tokens from '../data/tokens.json';
+
+export default function filterTokens(prefix: string) {
+  return Object.entries(tokens)
+    .filter(([name]) => name.startsWith(prefix))
+    .map(([name, value]) => ({
+      name: `--${name}`,
+      value,
+    }));
+}
```
Comment (dierat) [inline .storybook/util/filterTokens.ts:10]: Nice deduplication! ❤️

File: .storybook/components/DesignTokens/Tier1/TypographyPresets.js [hunk 4]
```diff
@@ -1,161 +1,60 @@
...
-          variant="typography-body"
-        />
-
-        <TokenSpecimen
-          name="eds-typography-preset-006"
-          specimenClassName={styles['u-typography-preset-006']}
-          variant="typography-body"
-        />
-
+    const renderTypeToken = (size, value) => {
```
Comment (dierat) [inline .storybook/components/DesignTokens/Tier1/TypographyPresets.js:8]: Nice!

File: .storybook/components/DesignTokens/Tier1/TypographyPresets.js [hunk 5]
```diff
@@ -1,161 +1,60 @@
...
+    };
 
-        <TokenSpecimen
-          name="eds-typography-preset-007"
-          specimenClassName={styles['u-typography-preset-007']}
-          variant="typography-body"
-        />
+    return (
+      <Grid>
+        {renderTypeToken('001', '40 / 48')}
```
Comment (dierat) [inline .storybook/components/DesignTokens/Tier1/TypographyPresets.js:22]: I really like that this exposes the underlying values so it's easier to compare to a design or an existing feature and make sure you're using the right typography level, but I'm also wondering if this is going to be hard to keep in sync with the tokens if we ever make changes to the actual typography styles 🤔 No suggestion; just putting it out there.

**Review by dierat — APPROVED** [review]

File: .storybook/components/DesignTokens/Tier1/TypographyPresets.js (hunk 5 above)
Comment (anniehu4) [reply .storybook/components/DesignTokens/Tier1/TypographyPresets.js:22]: yeah, great point 😬  let me see if we can grab the font size token values, but line height we're not exposing as tokens at all currently

**Pull request merged**